src/components/InstallTabs/index.tsx

```diff
@@ -64,6 +65,11 @@ const InstallTabs = (): JSX.Element => {
     }
   }
 
+  // if userOS doesn't match for some reason, return an empty element
+  if (systems[userOS] === undefined) {
```

Comment:
Wouldn't this already be using the `default` in the case statement?

Reply:
revamped logic